libavfilter/vf_drawbox.c

```diff
@@ -140,6 +143,19 @@ static void draw_region(AVFrame *frame, DrawBoxContext *ctx, int left, int top,
     }
 }
 
+static int box_source_string_parse(const char *box_source_string, enum AVFrameSideDataType *box_source)
+{
```

Comment:
simpler to return AVFrameSideDataType.

av_assert0(box_source_string)  because it is always required.

My comment of the function separate_name is not correct for the NULL checking.